back/backend/models/wca/round.py

```diff
@@ -0,0 +1,18 @@
+from google.cloud import ndb
+
+from backend.models.wca.base import BaseModel
+
+
+class RoundType(BaseModel):
+    rank = ndb.IntegerProperty()
+    name = ndb.StringProperty()
+    final = ndb.BooleanProperty()
```

Comment:
If possible, I would call this `is_final` since it's a boolean.

Reply:
Technically I could, but I would need to reload the whole wca db. I don't know yet if it's worth it.

Reply:
Ok leave it for now then. We can possibly do a migration in the future to rename the properties with better naming conventions.

Reply:
Actually thinking more about it I think making the change right now is smart. Because we only have the staging site up so it's less risky than if it was already in production.